roles/bridge/vars/ufw_rules.yml

```diff
@@ -0,0 +1,14 @@
+---
+bridge_ufw_rules:
+  validator:
+    - port: "{{ bridge_p2p_port }}"
+      proto: tcp
+      rule: allow
+      from: any
+      comment: p2p_port
+
+    - port: "{{ bridge_rpc_port }}"
+      proto: tcp
```

Comment:
rpc is not required

Reply:
we are running it on

[RPC]
Address = "localhost"

so rpc wont be open, so no ufw rule is required